src/chains/tests/test_models.py

```diff
@@ -10,3 +13,28 @@ def test_str_method_outputs_name_chain_id(self):
             str(chain),
             f"{chain.name} | chain_id={chain.id}",
         )
+
+
+class ChainColorValidationTestCase(TransactionTestCase):
+    def test_invalid_text_colors(self):
+        param_list = ["aaa", "bbb", "#fffffffff", "zzz", "010", "", "a word", "#hhh"]
+        for invalid_color in param_list:
+            with self.subTest(msg=f"Invalid color {invalid_color} should throw"):
```

Comment:
`subTest` will generate a new test for each param in `param_list`. This had some problems while running with a normal `TestCase` with the rollbacks (probably due to the way the context of the `subTest` is set). Running with a `TransactionTestCase` works fine

Reply:
I didn't know about `subTest`, but maybe in this case `@pytest.mark.parametrize` is more intuitive, what do you think? I need to dig on `pytest` as previously I was using regular python testing and there are some interesting things

Reply:
Unfortunately `@pytest.mark.parametrize` seems to not be compatible with Django Test Cases. I was able to make it work with a function test with access to DB via `@pytest.mark.django_db`. So the following works:

```python
@pytest.mark.django_db
@pytest.mark.parametrize
def test_invalid_text_colors(self):
```

But then I loose the assertion functions (like `assertRaises`). 😔

Reply:
Good to know 👍🏻